addons/event/models/event_registration.py

```diff
@@ -339,8 +340,8 @@ def _message_post_after_hook(self, message, msg_vals):
 
     def get_date_range_str(self):
         self.ensure_one()
-        today = fields.Datetime.now()
-        event_date = self.event_begin_date
+        today = pytz.utc.localize(fields.Datetime.now()).astimezone(pytz.timezone(self.event_id.date_tz))
```

Comment:
As the variables are now timezoned, I would add a `..._tz` at the end of the variables' name